app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,17 @@
+class LikesController < ApplicationController
+  def create
+    user = User.find(params[:user_id])
+    post = Post.find(params[:post_id])
+
+    like = Like.new(author_id: user.id, post_id: post.id)
+    if like.save && post.update_attribute(:likes_counter, post.likes_counter + 1)
```

Comment:
- [ ] <b>OPTIONAL:</b> You can implement the same suggestion I gave on post_count.